dvc/repo/metrics/show.py

```diff
@@ -151,6 +153,20 @@ def _read_metrics(self, metrics, branch):
     return res
 
 
+def _check_file_type(path, typ):
+    if path.endswith(".json"):
```

Comment:
Considering the current way we handle types https://github.com/iterative/dvc/blob/0.33.1/dvc/repo/metrics/show.py#L65 , looks like we could simplify these `if`s with a simple:
```
os.path.splitext(path.lower())[1]
```

Reply:
@cand126 , I agree with @efiop , you could have something like:
```python
# NOTE: Using the file extension to detect the type
typ = os.path.splitext(path.lower())[1].replace('.', '')
```